osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -132,12 +141,12 @@ def job_status(self, extraction_order):
                         {
                             "format": "fgdb",
                             "progress": "successful",
-                            "result_url": "http://localhost:8000/api/gis_format/11/download_result/"
+                            "result_url": "http://<your_ip>:8000/api/gis_format/11/download_result/"
```

Comment:
It's just an example, but as we're changing it here, anyway, shouldn't we change it to something like `http://<conversion service host>:8901/api/gis_format/11/download_result/`?